redfish-core/lib/pcie_slots.hpp

```diff
@@ -29,6 +31,55 @@
 namespace redfish
 {
 
+inline void
+    addLinkedPcieDevices(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                         const std::string& slotPath, size_t index)
+{
+    // Collect device associated with this slot and
+    // populate it here
+    constexpr std::array<std::string_view, 1> interfaces = {
+        "xyz.openbmc_project.Inventory.Item.PCIeDevice"};
+    dbus::utility::getSubTree(
+        slotPath, 0, interfaces,
```

Comment:
I have followed the logic from 1060, but yes, it should be at depth from the current tree.

However, this logic will be changed to use `containing/contained_by` associations once https://jsw.ibm.com/browse/PFEBMC-2479 is done.

So, I'm holding this PR for a while.

Reply:
Yes, I'll  first update the depth to 1 as like https://github.com/ibm-openbmc/bmcweb/pull/872

This will still wait for https://jsw.ibm.com/browse/PFEBMC-2479 and update to use `getAssociatedSubTree()` with the association